### PR TITLE
f-registration@v0.7.1: Add whitespace between preLinkText and link to remove reliance on translated string. 

### PR DESCRIPTION
**Pull request by luke-andrew-parry**
### Changed
- Added space between login preLinkText and linkText.

### CODE REVIEW

File: packages/f-registration/src/components/Registration.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
             v-if="shouldShowLoginLink"
             :class="$style['c-loginLink']"
             data-test-id="create-account-login-link">
-            {{ loginSettings.preLinkText }}<a :href="loginSettings.url">{{ loginSettings.linkText }}</a>
+            {{ loginSettings.preLinkText }}<a :href="loginSettings.url"> {{ loginSettings.linkText }}</a>
```
Comment (ashleynolan) [inline packages/f-registration/src/components/Registration.vue]: It makes more sense to me to add the space outside the `a` tag? (to the left of it)

Comment (luke-andrew-parry) [reply packages/f-registration/src/components/Registration.vue]: Done.

**Review by ashleynolan — APPROVED** [review]

File: packages/f-registration/src/components/Registration.vue [hunk 2]
```diff
@@ -10,7 +10,7 @@
             v-if="shouldShowLoginLink"
             :class="$style['c-loginLink']"
             data-test-id="create-account-login-link">
-            {{ loginSettings.preLinkText }}<a :href="loginSettings.url">{{ loginSettings.linkText }}</a>
+            {{ loginSettings.preLinkText }} <a :href="loginSettings.url">{{ loginSettings.linkText }}</a>
```
Comment (kevinrodrigues) [inline packages/f-registration/src/components/Registration.vue:13]: Have you tried to wrap this in a template or span? I think Vue preserves the spaces if you do that.

```javascript
<span>{{ loginSettings.preLinkText }}</span>
<a :href="loginSettings.url">{{ loginSettings.linkText }}</a>
```

**Review by smarkey — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

File: packages/f-registration/src/components/Registration.vue (hunk 2 above)
Comment (luke-andrew-parry) [reply packages/f-registration/src/components/Registration.vue:13]: I doesn't look to help here. I didn't want to rely on the translated text that gets passed in to have a space at the end to separate the`preLinkText` from the `linkText`.

**Pull request merged**